Hero.cpp

```diff
@@ -0,0 +1,16 @@
+#include "Hero.h"
+
+const void Hero::attackEnemy(Hero& h)

```

Comment:
A const-olas tovabbra sincs rendben, mi ertelme a const void visszateresi erteknek, mikor lattatok tolunk ilyet? A fuggvenynek kellene const-nak lenni.

Reply:
Illetve jol latom, hogy ez most nem meguti a `h`-t, hanem a `h` ut meg minket? Akkor a fuggveny nevet is erdemes lenne lecserelni, mert igy felrevezeto. Es ebben az esetben nyilvan nyilvan akkor nem a fuggveny const, hanem a tamado.

Reply:
Jogos volt a komment, közben változott a logikája a függvénynek, viszont a névvel nem követtük ezt. Most már a getAttack név nem megtévesztő, mint csúnyán fogalmazva "attackot gettel a hero". Const pedig javítva.